package.json

```diff
@@ -44,17 +44,19 @@
   "dependencies": {
     "iframe-resizer": "^3.5.0",
     "inherits": "^2.0.1",
-    "lie": "^3.0.2",
```

Comment:
Removed the Promise polyfill while I was in here, since IE11 is no longer a thing.